src/test/java/com/wikia/webdriver/testcases/discussions/UploadingImageTests.java

```diff
@@ -0,0 +1,264 @@
+  @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+  @Execute(onWikia = DESKTOP_COMMUNITY, asUser = User.USER_3)
+  public void userCannotUploadUnsupportedImageToTheirPostOnDesktop() {
+    PostsListPage page = new PostsListPage().open();
+    BasePostsCreator postCreator = startPostCreationDesktop(page);
+    String errorMsg = postCreator.uploadUnsupportedImage();
+    Assertion.assertStringContains(errorMsg, UNSUPPORTED_IMAGE_MSG);
+    postCreator.clickSubmitButton();
+    page.waitForPageReload();
+    Assertion.assertFalse(page.getPost().firstPostHasImage(), POST_UNSUPPORTED_IMAGE_NOT_VISIBLE);
+  }
+
+  @Test(groups = DESKTOP)
+  @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+  @Execute(onWikia = DESKTOP_COMMUNITY, asUser = User.USER_3)
+  public void userCannotUploadUnsupportedImageToTheirReplyOnDesktop() {
+    PostDetailsPage page = new PostDetailsPage().open(setUp(MOBILE_COMMUNITY).getId());
+    BaseReplyCreator replyCreator = startReplyCreationDesktop(page);
+    String errorMsg = replyCreator.uploadUnsupportedImage();
+    Assertion.assertStringContains(errorMsg, UNSUPPORTED_IMAGE_MSG);
+    replyCreator.clickSubmitButton();
+    page.waitForPageReload();
+    Assertion.assertFalse(page.findNewestReply().hasImage(), REPLY_UNSUPPORTED_IMAGE_NOT_VISIBLE);
+  }
+
+  @Test(groups = DESKTOP)
+  @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+  @Execute(onWikia = DESKTOP_COMMUNITY, asUser = User.USER_3)
+  public void userCanRemoveImagePreviewFromPostDraftOnDesktop() {
+    PostsListPage page = new PostsListPage().open();
+    startPostCreationDesktop(page).uploadImage().removeImage().clickSubmitButton();
+    page.waitForPageReload();
+    Assertion.assertFalse(page.getPost().firstPostHasImage(), POST_DELETED_IMAGE_NOT_VISIBLE);
+  }
+
+  @Test(groups = DESKTOP)
+  @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+  @Execute(onWikia = DESKTOP_COMMUNITY, asUser = User.USER_3)
+  public void userCanRemoveImagePreviewFromReplyDraftOnDesktop() {
+    PostDetailsPage page = new PostDetailsPage().open(setUp(MOBILE_COMMUNITY).getId());
+    startReplyCreationDesktop(page).uploadImage().removeImage().clickSubmitButton();
+    page.waitForPageReload();
+    Assertion.assertFalse(page.findNewestReply().hasImage(), REPLY_DELETED_IMAGE_NOT_VISIBLE);
+  }
+
+  @Test(groups = DESKTOP, enabled = false)
+  @RelatedIssue(issueID = "IRIS-4896", comment = "To be implemented")
+  @Execute(onWikia = DESKTOP_COMMUNITY, asUser = User.USER_3)
+  @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+  public void userCanOverwriteOpenGraphImageInExistingPostWithUploadedImageOnDesktop() {
+
+  }
+
+  @Test(groups = DESKTOP, enabled = false)
+  @RelatedIssue(issueID = "IRIS-4896", comment = "To be implemented")
+  @Execute(onWikia = DESKTOP_COMMUNITY, asUser = User.USER_3)
+  @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+  public void userCanOverwriteOpenGraphImageInExistingReplyWithUploadedImageOnDesktop() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1186)